src/twisted/scripts/trial.py

```diff
@@ -649,13 +642,13 @@ def run():
 
     suite = _getSuite(config)
     if config["until-failure"]:
-        test_result = trialRunner.runUntilFailure(suite)
+        testResult = trialRunner.runUntilFailure(suite)
```

Comment:
ir looks like we have a missing automated tests here.

I think is ok to merge this without adding a new test for this change, as the change is minor.